### PR TITLE
Solvathon Update - InnoFest webpage

### PR DESCRIPTION
**Pull request by MakiseKurisuX**
## Description

InnoFest page update - adding further Solvathon description

Working as Intended

## Screenshots / Videos

https://i.gyazo.com/481a63e5b1e590e337b55d0a04f273f7

### CODE REVIEW

File: pages/innofest/index.tsx [hunk 1]
```diff
@@ -80,23 +86,142 @@ export default function InnoFest(props: LandingBodyProps) {
...
+                  </ListItem>
+                  <ListItem>
+                    <ListIcon as={ChevronRightIcon} color={nGreen[400]} />
+                    <b className="glowtext2">6 Feb 2023</b> onwards: Lookout for the <Text as="span" color={nYellow[400]}>SOLVATHON</Text>{" "} poll
+                    and up-vote the entry you want to see SOLVED in the year ahead.
+                  </ListItem>
+                  <ListItem>
+                    <ListIcon as={ChevronRightIcon} color={nGreen[400]} />
+                    <b className="glowtext2">23 Feb 2023</b>: Top 3 voted entries stand to win attractive prizes at InnoFest!
+                  </ListItem>
```
Comment (ryanntannn) [inline pages/innofest/index.tsx:180]: can consider using `.map` for repeated components

File: pages/innofest/index.tsx [hunk 2]
```diff
@@ -80,23 +86,142 @@ export default function InnoFest(props: LandingBodyProps) {
...
+              id="solvathon"
+              mb={5}
+              >
+                <Heading size={"lg"} mb={5} color={nYellow[400]} className="glowtext3">
+                  What you need to do?
+                </Heading>
+                <List spacing={3} >
+                  <ListItem>
+                    <ListIcon as={ChevronRightIcon} color={nGreen[400]} />
+                    Bookmark <a href="http://localhost:3000/innofest">this URL</a> or join our SW<i>i</i>FT Telegram Channel.
```
Comment (ryanntannn) [inline pages/innofest/index.tsx:157]:
```suggestion
                    Bookmark <Link href="/innofest">this URL</Link> or join our SW<i>i</i>FT Telegram Channel.
```
the localhost url will not work for prod and stag env where the url is different, consider using mui's `Link` component as well

**Review by ryanntannn — CHANGES_REQUESTED** [review]
LGTM visually, tested mobile responsiveness. 
Added comment on one of the urls that contains localhost url that will break in prod
Added comments on avoiding using css and using .map for ur learning.

File: pages/innofest/index.tsx (hunk 2 above)
Comment (MakiseKurisuX) [reply pages/innofest/index.tsx]: Done with this

**Review by ryanntannn — APPROVED** [review]

**Pull request merged**